src/arrayMethodReduce.js

```diff
@@ -5,8 +5,20 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
+    let strartIndex = 0;
+
+    if (arguments.length < 2) {
```

Comment:
```suggestion
    if (initialValue === undefined) {
```
Було б логічно перевіряти саме на наявність initialValue, а не на кількість аргументів

Reply:
![image](https://user-images.githubusercontent.com/26704490/138920662-f10618b9-f633-47fc-af5e-a6370361a5ac.png)
при таком условии не проходит один тест(